native/acter/api.rsh

```diff
@@ -217,6 +217,9 @@ object OptionBuffer {
 object OptionRsvpStatus {
     /// get status
     fn status() -> Option<RsvpStatus>;
+
+    /// get status in string
+    fn status_str(pascal_case: bool) -> Option<string>;
```

Comment:
why do we want this to have pascal_case support? It doesn't make the mapping easier either ... Why not just map on `'yes', 'no' and 'maybe'` to generate the dart enum?

Reply:
`pascal_case` is used [here](https://github.com/bitfriend/a3/blob/25f8077dc1591013109cf1f9a2c671746be6edfa/app/lib/features/events/widgets/events_item.dart#L92).

Reply:
okay, the code is like this now, but we shouldn't really do that. Just think of translations and it becomes clear that we can't just use the string-representation of the enum-case and show that. So we should rather remap it to the proper enum case and then only use that enum on the dart-side ...